examples/complete-web/data.tf

```diff
@@ -0,0 +1,23 @@
+data "aws_availability_zones" "available" {}
+data "aws_caller_identity" "current" {}
+
+data "aws_ami" "amazon_linux_ecs_generic" {
+  most_recent = true
+
+  owners = ["amazon"]
+
+  filter {
+    name   = "name"
+    values = ["amzn2-ami-ecs-hvm-*-x86_64-ebs"]
+  }
+
+  filter {
+    name   = "owner-alias"
+    values = ["amazon"]
+  }
+}
+
```

Comment:
This seems not to be required as we don't use ec2 s
```suggestion
```

Reply:
@AutomationD we use it as image id for ECS-APP